ui/src/literature/components/ReferenceItem.jsx

```diff
@@ -64,6 +68,14 @@ class ReferenceItem extends Component {
       'collaborations_with_suffix'
     );
 
+    const displayEditButtonV1 = getConfigFor(
+      'SELF_CURATION' && 'SELF_CURATION_BUTTON_V1'
```

Comment:
hmmm this statement will return `SELF_CURATION_BUTTON_V1`, why we need the `SELF_CURATION`?